arch/armv8/rpi3/CMakeLists.compiler

```diff
@@ -0,0 +1,21 @@
+#toolchains
+INCLUDE(CMakeForceCompiler)
+
+# this one is important
+SET(CMAKE_SYSTEM_NAME Generic)
+SET(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
+
+# which compilers to use for C and C++
+
+find_program(CMAKE_ASM_COMPILER aarch64-linux-gnu-gcc)
+find_program(CMAKE_C_COMPILER aarch64-linux-gnu-gcc)
+find_program(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
```

Comment:
Using a compiler that targets linux could be problematic. `aarch64-none-elf-gcc` would probably be better.